### ISSUE TITLE
Lif getaction

### ISSUE THREAD

**Issue by MKaptein** [MKaptein/streamingbandit#72]
In the following get action code:

```
safe_builtins = {'print' : print, 'base' : base, 'numpy' : np, 'np' : np, 'scipy' : sp, 'sp' : sp, 'bts': bts, 'lm' : lm, 'lif': lif, 'thompson' : thompson, 'random' : random}
theta = self.get_theta(all_float=False, key="treatment")
Lif = lif.Lif(theta, x0=3.0, a=.5 , t=20, gamma=.2, omega=1.0, lifversion=1)
suggestion = Lif.suggest()
self.action["x"] = suggestion["x"]
self.action["t"] = suggestion["t"]
self.set_theta(Lif, key="treatment")
```
The last line should not be necessary.

**Comment by robinvanemden** [comment]
Set theta was indeed superfluous. I was able to remove it by keeping score of t and x0 in the lib.

**Comment by g0ulash** [comment]
Fixed in 6fe9c1b. Also changed default code.
